app/views/brands/explore.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1>Explore eco-brands</h1>
+
+<h2>Brands</h2>
+<p>(waiting for seed)</p>
+<% @brands.each do |brand| %>
+    <%=brand.splash_image%>
+    <h2><%=brand.name%></h2>
```

Comment:
@flaminiavg this is all great 🥳 do you also want to add brand average rating and brand price rating here too, to account for the hover state 👇

<img width="311" alt="Screenshot 2021-11-23 at 14 42 10" src="https://user-images.githubusercontent.com/80174279/143045062-3e62e080-04f7-4cde-bf30-28dfc1a6f950.png">
🏼

Reply:
Thank you for the reminder!! I just added the price category and the av. rating. For the moment the rating is commented out (waiting for emily's method), but it works on the console:)